### PR TITLE
Feature/clear grid

### PR DESCRIPTION
**Pull request by dcochico**
## What I did:
- Add key to Nav component
- Update styling to Nav buttons
- Add logic for clearing garden, but commented everything out until JSON contract is ready.

## Issues closed:
- #21
- #53
- #54

## Did this break anything?
- [x] No
- [ ] Yes

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor(DRY-ing up/ reorganizing code, etc.)
- [x] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [x] There are comments where clarification/ organization is needed
- [ ] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

## What's next?
- Would like to meet with FE and BE to confirm the body of the PATCH request
- Would like to collaborate on drag-and-drop functionality

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link href='https://fonts.googleapis.com/css?family=Rajdhani' rel='stylesheet'>
```
[the remark this answers is not in the document]
Comment (dcochico) [reply public/index.html:5]: just added this in my latest commit! thanks for the suggestion.

**Comment by dcochico** [comment]
> Could be cool to have two clear buttons - one that resets the garden back to a plain grid with only available cells, and one that clears only unplanted cells, leaving disabled and planted cells as they are

just added this in my latest commit, thanks for the suggestion!

File: src/components/List/List.scss [hunk 2]
```diff
@@ -30,12 +31,13 @@
...
+  // min-width
   cursor: pointer;
 }
 
 .list-container {
   background-color: #beab95;
-  height: 750px;
-  width: 750px;
+  height: calc(100vh - 125px);
+  width: calc(100vh - 125px);
```
Comment (mbenfowler) [inline src/components/List/List.scss:41]: niice catch, we'll likely need to update these dimensions again to accommodate the drag-and-drop resizing we did

File: src/components/Nav/Nav.tsx [hunk 3]
```diff
@@ -1,31 +1,53 @@
...
 const Nav = ({ isGardenView, setIsGardenView }: NavProps) => {
   const [popUp, setPopUp] = useState<boolean>(false);
+  const [fullClear, setFullClear] = useState<boolean>(false);
+  const [partialClear, setPartialClear] = useState<boolean>(false);
+
+  const reset = () : void => {
+    setPopUp(false);
+    setFullClear(false);
+    setPartialClear(false);
+  }
```
Comment (mbenfowler) [inline src/components/Nav/Nav.tsx:22]: niice

**Review by mbenfowler — APPROVED** [review]
i got nothin, great job

**Pull request merged**